common/tools/dev-tool/src/util/testProxyUtils.ts

```diff
@@ -172,7 +172,7 @@ function runCommand(executable: string, argv: string[], options: SpawnOptions =
 }
 
 export async function runTestProxyCommand(argv: string[]): Promise<void> {
-  const result = runCommand(await getTestProxyExecutable(), argv, { stdio: "inherit" }).result;
+  const result = runCommand(await getTestProxyExecutable(), argv, { stdio: "inherit", env: { ...process.env } }).result;
```

Comment:
Nit: according to the [child_process docs](https://nodejs.org/api/child_process.html#child_processspawncommand-args-options) `env` defaults to `process.env`. Is this change necessary?

Reply:
we need to pass the parent's env variables though.